code_sharing/Makefile

```diff
@@ -0,0 +1,22 @@
+# LOCAL / DOCKER
+run-server:
+	# Depends on `make build`
+	docker-compose up -d
+
+stop-server:
+	docker-compose stop
+
+# Flutter
+flutter-local:
+	cd client && flutter run -d macos
```

Comment:
I feel like we should set this up for linux as well. 

And then there is Windows...

Reply:
The `Makefile` may actually just be a bad idea 🤔